src/openms/source/APPLICATIONS/ConsoleUtils.cpp

```diff
@@ -121,6 +120,7 @@ namespace OpenMS
         {
           LOG_DEBUG << "output shaping: stty size command failed." << std::endl;
         }
+        pclose(fp);
```

Comment:
It should return -1 then, if I understand [pclose](http://pubs.opengroup.org/onlinepubs/009695399/functions/pclose.html) correctly. Branching would, if I understand you correctly, still raise an issue with coverity.